stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -665,6 +680,23 @@ impl<'a> ChainstateTx<'a> {
             }
         }
     }
+
+    pub fn get_index_block_hashes_from_txid(
```

Comment:
The only way this would return multiple rows is if a transaction was mined in two different blocks.  Is that a query this function is intended to serve?

Reply:
Well the same txid can ends multiple times in the transactions table, obviously we are interested in txid that actually ends in the chain so it will be a 2 step process where we return only the row with a valid (in the staging db) index_block_hash. If a user knows the index_block_hash this would not be required, but at the same time if the index-block_hash is known there is no need for this endpoint (you can just extract the transaction from the block)

Reply:
> Well the same txid can ends multiple times in the transactions table,

Can you explain to me how this happens?